wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,162 @@
+<template lang='pug'>
+  .container
+      .container-register
+        img.logo(alt='Wolox logo' src='../assets/logo.png')
+        form(@submit.prevent='onSubmit').container-form
+          .container-field(v-for='(field, name) in fields' :key='name')
```

Comment:
Maybe instead of the `v-for` it's more convenient to have them in the template. What do you think? 😄